projects/Mallard/android/build.gradle

```diff
@@ -11,6 +11,8 @@ buildscript {
         supportV4Version = '1.0.0' // Do not specify if using old libraries, specify '1.0.0' or similar for androidx.legacy:legacy-support-v4 dependency
         googlePlayServicesVersion = "17.1.0"
         firebaseVersion = "19.0.1"
+        androidXAnnotation = "1.1.0"
```

Comment:
Worries me...

Reply:
It does build 😬 - is there anything else that could go wrong at runtime rather than compile time?